controllers/grid/queries/QueriesGridHandler.php

```diff
@@ -664,11 +664,15 @@ public function updateQuery($args, $request)
                     continue;
                 }
                 $submission = $this->getSubmission();
-                $mailable = $this->getStageMailable($request->getContext(), $submission);
-                $mailable->sender($currentUser);
-                $mailable->body($note->getData('contents'));
-                $mailable->subject($note->getData('title'));
-                $mailable->recipients([$user]);
+
+                $context = $request->getContext();
+                $mailable = $this->getStageMailable($context, $submission, $note->getData('title'), $note->getData('contents'));
```

Comment:
Is the user able to submit the new query form with an empty title? If so, can we add a test for that?

Reply:
No, in the discussions/queries grid the subject is required. In participants grid the subject is retrieved from a template or by default from the NOTIFICATION template